spec.md

```diff
@@ -1135,10 +1129,9 @@ message GetNodeIDRequest {
 }
 
 message GetNodeIDResponse {
-  // The ID of the node which SHALL be used by CO in
-  // `ControllerPublishVolume`. This is an OPTIONAL field. If unset,
-  // the CO SHALL leave the `node_id` field unset in
-  // `ControllerPublishVolume`.
+  // The ID of the node as understood by the SP which SHALL be used by
+  // CO in subsequent `ControllerPublishVolume`.
+  // This is a REQUIRED field.
```

Comment:
suggestion: This is an OPTIONAL field that is REQUIRED only if the controller advertises the `PUBLISH_UNPUBLISH_VOLUME` capability.

because not implementing the call means that GetNodeIDResponse is empty, and that violates the REQUIRED semantics as currently documented.

Reply:
https://github.com/container-storage-interface/spec/pull/152